health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/LocalisationEditorPopup.js

```
@@ -0,0 +1,107 @@
+            }))
+          }
+          placeholder={t("ENTER_TRANSLATION")}
+        />
+      ),
+      width: "55%",
+    },
+  ];
+  
+  return (
+    <div style={{ minWidth: "min-content" }}>
+      <div style={{ display: "flex", justifyContent: "space-between", marginBottom: "1rem" }}></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)